azurerm/resource_arm_private_link_service_test.go

```diff
@@ -0,0 +1,676 @@
+
+  nat_ip_configuration {
+    name                         = "primaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.17"
+    private_ip_address_version   = "IPv4"
+    primary                      = true
+  }
+
+  nat_ip_configuration {
+    name                         = "secondaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.20"
+    private_ip_address_version   = "IPv4"
+    primary                      = false
+  }
+
+  nat_ip_configuration {
+    name                         = "thirdaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.19"
+    private_ip_address_version   = "IPv4"
+    primary                      = false
+  }
+
+  nat_ip_configuration {
+    name                         = "fourtharyIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.18"
+    private_ip_address_version   = "IPv4"
+    primary                      = false
+  }
+
+  load_balancer_frontend_ip_configuration_ids = [
+    azurerm_lb.test.frontend_ip_configuration.0.id
+  ]
+
+  tags = {
+    env = "test"
+  }
+}
+`, testAccAzureRMPrivateLinkServiceTemplate(rInt, location), rInt, rInt, rInt, rInt, rInt)
+}
+
+func testAccAzureRMPrivateLinkService_complete(rInt int, location string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_private_link_service" "test" {
+  name                           = "acctestpls-%d"
+  location                       = azurerm_resource_group.test.location
+  resource_group_name            = azurerm_resource_group.test.name
+  auto_approval_subscription_ids = [data.azurerm_subscription.current.subscription_id]
+  visibility_subscription_ids    = [data.azurerm_subscription.current.subscription_id]
+
+  nat_ip_configuration {
+    name                         = "primaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.17"
+    private_ip_address_version   = "IPv4"
+    primary                      = true
+  }
+
+  nat_ip_configuration {
+    name                         = "secondaryIpConfiguration-%d"
+    subnet_id                    = azurerm_subnet.test.id
+    private_ip_address           = "10.5.1.18"
+    private_ip_address_version   = "IPv4"
+    primary                      = false
+  }
+
+  load_balancer_frontend_ip_configuration_ids = [
+    azurerm_lb.test.frontend_ip_configuration.0.id
+  ]
+
+  tags = {
+    env = "test"
+  }
+}
+`, testAccAzureRMPrivateLinkServiceTemplate(rInt, location), rInt, rInt, rInt)
+}
+
+func testAccAzureRMPrivateLinkServiceTemplate(rInt int, location string) string {
+	return fmt.Sprintf(`
+data "azurerm_subscription" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
Could we change this to
```suggestion
  name     = "acctestRG-privatelink-%d"
```

Reply:
Fixed.